### PR TITLE
Update BDLaethd17.xml

### PR DESCRIPTION
**Pull request by gnizla**
Very small deco note, but note that I am proposing to shift not before dating from 1500 to 1650-1750 based on style of headband.

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethd17.xml [hunk 1]
```diff
@@ -424,6 +424,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     the punctuation marks, text dividers, and of numerals.</seg>
                             </handNote>
                         </handDesc>
+
+                        <decoDesc>
+                          <summary>The decorative frame on <locus target="#4r"/> marks
+                        the incipit and is painted in a style associated with works produced in the historical region
+                      of Lasta around <date>1700</date> by the manuscript's former owner <bibl><ptr target="bm:1977Juel-Jensen"/></bibl>
```
Comment (thea-m) [inline OxfordBodleian/Juel-Jensen/BDLaethd17.xml:431]: bibliographic reference between parantheses?

**Review by thea-m — CHANGES_REQUESTED** [review]
Excellent, thank you! I have of course nothing against the dating, on which I trust you completely (and I am very happy that this narrows the estimated date of production of the manuscript). I would like to propose to adapt the wording inside `<history>` slightly to reflect that the main evidence for this timespan is the frame, not the handwriting ("Based on the style of the decorative frame on f. ..., the date of production can be set to..." or whatever you find good). I think that being clear about the dating evidence is methodologically significant and interesting for readers. Thanks again :)

File: OxfordBodleian/Juel-Jensen/BDLaethd17.xml [hunk 2]
```diff
@@ -488,9 +503,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     </physDesc>
                     <history>
                         <origin>
-                            <origDate from="1500" to="1699" evidence="lettering"> </origDate> The
-                            earliest date of production—taking into account the handwriting—can
-                            be set to the sixteenth century. The name of <persName
+                            <origDate from="1650" to="1750" evidence="lettering"> </origDate> The
+                            earliest date of production—taking into account the handwriting—can and the frame on <locus target="#4r"/>
+                            be a period between the mid-seventeenth to mid-eighteenth century. The name of <persName
```
Comment (ElianaDalSasso) [inline OxfordBodleian/Juel-Jensen/BDLaethd17.xml:508]: -can and the frame....be a period
Should 'can' be shifted close to 'be'?

**Review by ElianaDalSasso — COMMENTED** [review]
Thank you Jacopo! Really interesting, I have just a minor comment.

**Pull request closed without merging**